CHANGELOG.md

```diff
@@ -25,6 +26,10 @@
 - dbaas: remove redis create
 - SKS cluster: display enable kube proxy #675
 
+### Improvements
+
+- SKS: related commands are migrated to egoscale v3
+
```

Comment:
I think rebases got that in the wrong spot

Reply:
@tgrondier what do you mean?